libexec/cli/pull.exec

```diff
@@ -86,7 +86,7 @@ case "$IMPORT_URI" in
     ;;
     shub://*)
         IMAGE_SHUB_ID=`echo "$IMPORT_URI" | sed -e 's@^shub://@@'`
-        exec $SINGULARITY_libexecdir/singularity/python/cli.py --shub $IMAGE_SHUB_ID
+        exec $SINGULARITY_libexecdir/singularity/python/cli.py --shub $IMAGE_SHUB_ID --pull-folder $PWD
```

Comment:
`--pull-folder` is an argument that specifies the output directory for the pull. For the first PR it was just pulling (downloading) to the cache, but then you mentioned that it was more intuitive for it to plop wherever the user is, and I definitely agree. I hard coded the $PWD, but again this variable could be some higher level argument that the user can control (for example, if they want it to go to a cache, or to some folder for their images, etc.)